conclave-common/src/main/kotlin/com/r3/conclave/common/internal/SgxTypes.kt

```diff
@@ -701,6 +705,9 @@ object SgxMetadataCssBuffer : Struct() {
     val q2 = field(FixedBytes(SE_KEY_SIZE))
 }
 
+/**
+ * This is also known as the `SIGSTRUCT`.
```

Comment:
Not sure. This maps to [`enclave_css_t`](https://github.com/intel/linux-sgx/blob/b0af6e75ac519ad5002a68fda0b672feddc1c92d/common/inc/internal/arch.h#L258). We should rename this to `SgxEnclaveCss` to be consistent, but this can be done in the other PR.